notebooks/15_flightplanning_tuto.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Flight Planning\n",
+    "\n",
+    "The problem we are targetting is to predict the trajectory of an aircraft under forecasted weather conditions.\n",
+    "\n",
+    "This problem has been chosen for two reasons:\n",
+    "  - Show how scikit-decide can be used in close to real life use case,\n",
+    "  - Design a specific domain using external packages. In our case, we will use [OpenAP](https://github.com/TUDelft-CNS-ATM/openap) to simulate the dynamics of the aircraft.\n",
+    "\n",
+    "Therefore in this notebook we will go through the following steps:\n",
+    "  - Wrap a Gym environment in a scikit-decide domain;\n",
+    "  - Use a classical A*Star algorithm to solve our problem;\n",
+    "  \n",
+    "Notes:\n",
+    "\n",
+    "- In order to focus on scikit-decide use, we put some code not directly related to the library in a separate module (weather data download and interpolation).\n",
+    "- This notebook can be long to run as it needs to retrieve a large amount of data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q --upgrade git+https://github.com/junzis/openap\n",
+    "!pip install -q --upgrade git+https://github.com/junzis/openap-top"
+   ]
+  },
```

Comment:
We need to think to add these to binder environment.

It seems that to be sure the pip install is done in the environment corresponding to the current python kernel, the most appropriate way to do it is:
```python
import sys
!{sys.executable} -m pip install -q --upgrade git+https://github.com/junzis/openap
!{sys.executable} -m pip install -q --upgrade git+https://github.com/junzis/openap-top
```

Reply:
We need also to add cartopy dependency. 
But the install needs also to other non-python dependencies GEOS, Shapely, and pyshp. These are installed directly if using the conda package but not with pip. 
So, for binder, we can use the conda package or (apt-get)
for local users (I mean users running the notebook locally) we need to add a comment specifying the need of cartopy but letting them installing their own way, with perhaps a link to the cartopy documentation. (Because a mere`pip install cartopy` does not work and lead to a not explicit error)

Reply:
Actually i did not succeed in installing cartopy in my python 3.7 conda environment with pip but succeed on a python 3.8 virtual environment. In any case, we must tell the user to install it but let them how to do it accorfing to their plateform.

Reply:
Et il y aussi
- cdsapi
- cfgrib: qui m'a nécessité `sudo apt install libeccodes-tools`
- pygeodesy